contrib/mesos/pkg/minion/task.go

```diff
@@ -0,0 +1,519 @@
+/*
```

Comment:
I had the same thought. I wasn't sure that I had the interfaces quite right, so was holding off on that for now. Hence, everything is package private. I'm pretty comfortable with the task stuff living here ATM, especially since we do define `interface` types to help decouple things.